chapters/planning-data-work.tex

```diff
@@ -1,147 +1,457 @@
-%------------------------------------------------
+% ----------------------------------------------------------------------------------------------
 
 \begin{fullwidth}
-Preparation for data work begins long before you collect any data.
-In order to be prepared to work on the data you receive, 
-you need to know what you are getting into.
+Preparation for collaborative data work begins long before you collect any data,
+and involves planning both the software tools you will use yourself
+and the collaboration platforms and processes for your team.
+In order to be prepared to work on the data you receive with a group,
+you need to plan out the structure of your workflow in advance.
 This means knowing which data sets and output you need at the end of the process,
-how they will stay organized and linked,
-what different types and levels of data you'll handle,
-and whether the data will require special handling due to volume or privacy considerations.
+how they will stay organized, what types of data you'll handle,
+and whether the data will require special handling due to size or privacy considerations.
 Identifying these details creates a \textbf{data map} for your project,
 giving you and your team a sense of how information resources should be organized.
 It's okay to update this map once the project is underway --
 the point is that everyone knows what the plan is.
 
-Then, you must identify and prepare your tools and workflow.
-Changing software and protocols half-way through a project can be costly and time-consuming,
-so it's important to think ahead about decisions that may seem of little consequence
-(think: creating a new folder and moving files into it).
-This chapter will discuss some of often overlooked tools and processes that
+To implement this plan, you will need to prepare collaborative tools and workflows.
+Changing software or protocols halfway through a project can be costly and time-consuming,
+so it's important to think ahead about decisions that may seem of little consequence.
+For example, things as simple as sharing services, folder structures, and filenames
+can be extremely painful to alter down the line in any project.
+Similarly, making sure to set up a self-documenting discussion platform
+and version control processes
+makes working together on outputs much easier from the very first discussion.
+This chapter will discuss some tools and processes that
 will help prepare you for collaboration and replication.
 We will try to provide free, open-source, and platform-agnostic tools wherever possible,
```

Comment:
I know we are only saying we will _try_ to provide tools that are free and open source, but the chapter is very Stata heavy with is very far from either of those. Can we mention the exception of Stata here? Otherwise this feels a bit bait-and-switchy.

Reply:
329275d4496f10e26fe2e2a3044ca213d58b3089